<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>979</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_new">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_import">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>0</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Import file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_bigText">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_smallText">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save file</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>971</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>401</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Set front</string>
    </property>
    <widget class="QComboBox" name="comboBox_bg">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>white</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_sl">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>white</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_tx">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>black</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>white</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>green</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>gray</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>blue</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>120</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Background:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Selection:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Text:</string>
     </property>
    </widget>
    <widget class="QFontComboBox" name="fontComboBox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>171</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>979</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
